spec/services/data_migrations/fix_candidate_api_updated_at_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe DataMigrations::FixCandidateAPIUpdatedAt do
+  around do |example|
+    Timecop.freeze { example.run }
+  end
+
+  it 'does not change valid `candidate_api_updated_at` values' do
+    valid_candidate = create(
+      :candidate,
+      candidate_api_updated_at: 5.days.ago,
```

Comment:
Should we set this here if we're overriding below?

Reply:
Good spot! Removed.